examples/presigned-getobject-request-date.js

```diff
@@ -27,8 +27,14 @@ var s3Client = new Minio.Client({
 })
 
 // Presigned get object URL for my-objectname at my-bucketname, it expires in 7 days by default.
-var requestDate = new Date('2017-01-01');
-var presignedUrl = s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, {}, requestDate, function(e, presignedUrl) {
+var requestDate = new Date('2017-01-01')
+s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, {}, requestDate, function(e, presignedUrl) {
   if (e) return console.log(e)
   console.log(presignedUrl)
 })
+
+// Versioning support
+s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, {versionId:"10fa9946-3f64-4137-a58f-888065c0732e"}, new Date('2017-01-01'), function(e, presignedUrl) {
```

Comment:
The same comment as before:
```suggestion
s3Client.presignedGetObject('my-bucketname', 'my-objectname', 1000, {versionId:"10fa9946-3f64-4137-a58f-888065c0732e"}, new Date(), function(e, presignedUrl) {
```